platforms/common/src/kernels/velocityVerlet.cc

```diff
@@ -226,58 +249,57 @@ extern "C" __global__ void integrateVelocityVerletPart3(int numAtoms, int numPai
             v2.x = comVel.x + relVel.x*mass1fract + (deltaXconstrained.x - v2.x*stepSize.y)*oneOverDt;
             v2.y = comVel.y + relVel.y*mass1fract + (deltaXconstrained.y - v2.y*stepSize.y)*oneOverDt;
             v2.z = comVel.z + relVel.z*mass1fract + (deltaXconstrained.z - v2.z*stepSize.y)*oneOverDt;
-#if __CUDA_ARCH__ < 130
+#ifdef SUPPORTS_DOUBLE_PRECISION
```

Comment:
And here.

Reply:
Oh wow, I got it wrong in all places 🤦‍♂